input/docs/getting-started/installation/index.md

```diff
@@ -75,9 +75,43 @@ ReactiveUI packages are now signed by the dotnet foundation. Only builds from th
 [ValidationsBadge]: https://img.shields.io/nuget/v/ReactiveUI.Validation.svg
 [ValidationsDocs]: https://reactiveui.net/docs/handbook/user-input-validation/
 
-
 > **Note** ReactiveUI has packages for older .NET versions. Those packages are unlisted from NuGet and not supported, but you can still use them at your own risk to have ReactiveUI running on good old devices, such as Lumias, Surface Hubs, Windows XP, etc. See [Delisting of versions before 8.0.0 from NuGet](https://reactiveui.net/blog/2018/05/delisting-of-versions-before-8-0-0-from-nuget) blog post for more info.
 
+## Example
+
+The following isn't an exhausitve example, as some solutions will follow a different project structure or only use a subset of the platforms, but it should be enough to help with which packages to install on each platform project.
+
+<pre>
+    .
+    ├── {NAME}.Tests.Unit
+        ├── ReactiveUI.Testing
+    ├── {NAME}.Core (.NetStandard)
+        ├── ReactiveUI
+        ├── ReactiveUI.Fody
+    ├── {NAME}.Droid
+        ├── ReactiveUI.AndroidSupport
```

Comment:
AndroidSupport is deprecated. AndroidX also exists now

Reply:
Yep. Not so familiar with Android... does this mean that AndroidX is now the mainstream version (Reactive.AndroidSupport is only still around to support older projects?)

Reply:
Basically that's where it's heading yeah. We support both at the moment.

Reply:
Is AndroidX supported in the core ReactiveUI package, or is another dependency required?

Reply:
https://www.nuget.org/packages/ReactiveUI.AndroidX/

Reply:
Added